compiler/wasm/src/compile.rs

```diff
@@ -136,6 +161,7 @@ pub fn compile(
     entry_point: String,
     contracts: Option<bool>,
     dependency_graph: Option<JsDependencyGraph>,
+    file_source_map: PathToFileSourceMap,
```

Comment:
We could make this a reference, so that Js code can pass multiple instances of this. The issue is that there needs to be a clone somewhere, so it may be better to make it explicit on the Js side

Reply:
Agreed, no sense in forcing an unnecessary clone in the case where we only pass one instance.